addons/markers/functions/fnc_updateMapData.sqf

```diff
@@ -0,0 +1,51 @@
+/*
+ * Author: commy2, SzwedzikPL
+ * Updates markers data on server
+ *
+ * Arguments:
+ * 0: Map for which markers are assigned to <STRING>
+ * 1: Markers data in format [newMarkersData, deletedMarkers] <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+if(!isServer) exitWith {};
+
+params ["_map", "_mapData"];
+TRACE_2("updateMapData",_map,_mapData);
+
+_mapData params ["_newMarkers", "_deletedMarkers"];
+
+private _mapID = _map call FUNC(getMapID);
+if (_mapID <= 0) exitWith {};
+
+private _mapMarkers = missionNamespace getVariable [format ["%1_%2", QGVAR(allMapMarkers), _mapID], []];
```

Comment:
I recommend storing this in a CBA namespace instead of on the missionNamespace. Just search `CBA_fnc_createNamespace` in the ACE codebase for examples.

Using a namespace makes prefixing `_mapId` unnecessary, so this could be:

```sqf
private _mapMarkers = GVAR(mapNamespace) getVariable _mapID;
if (isNil "_mapMarkers") then {
    _mapMarkers = []
};
```

(that if is needed because namespace do not support the getVariable ARRAY syntax)